barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.cpp

```diff
@@ -200,6 +200,10 @@ template <class ProverInstances>
 FoldingResult<typename ProverInstances::Flavor> ProtoGalaxyProver_<ProverInstances>::fold_instances()
 {
     BB_OP_COUNT_TIME_NAME("ProtogalaxyProver::fold_instances");
+    // Ensure instances are all of the same size
```

Comment:
This has bitten me a few times and usually verification just fails without any other asserts being triggered.